tests/test_freqslicer_broker.py

```diff
@@ -0,0 +1,257 @@
+import pytest
```

Comment:
AFAICT, this test doesn't do what I thought it was going to: check that the state that is registered in the broker is correct and that the dataset_id's the broker has are the ones that appear in the data. Is there a reason, it can't/doesn't do that one?

Reply:
This test is very old, I only reactivated it here. But I guess it was meant to be a high level test, as it doesn't do what you describe directly, but lets visWriter do it and then tests against what ends up on disk.